frontends/web/src/routes/settings/components/appearance/activeCurrenciesDropdownSetting.tsx

```diff
@@ -25,13 +26,20 @@ const ActiveCurrenciesDropdownSetting = () => {
   const { t, i18n } = useTranslation();
   const { activeCurrencies, defaultCurrency } = useContext(RatesContext);
   const { formattedCurrencies } = useLocalizedFormattedCurrencies(i18n.language);
+  const [isMobileDropdownOpen, setIsMobileDropdownOpen] = useState(false);
+  const isMobile = useMediaQuery('(max-width: 768px)');
+
   return (
     <SettingsItem
+      disabled={!isMobile}
       collapseOnSmall
       settingName={t('newSettings.appearance.activeCurrencies.title')}
       secondaryText={t('newSettings.appearance.activeCurrencies.description')}
+      onClick={() => setIsMobileDropdownOpen(true)}
       extraComponent={
         <ActiveCurrenciesDropdown
```

Comment:
Since this is not always a Dropdown now, what do you think about renaming to something else like Selector or Picker?

but it seems you'd need to rename a bit more than just the component i.e. isMobileDropdownOpen -> isMobileSelectorOpen etc.